script/zip_manifests/dist_zip.mac.x64.manifest

```diff
@@ -3,11 +3,14 @@ Electron.app/Contents/
 Electron.app/Contents/Frameworks/
 Electron.app/Contents/Frameworks/Electron Framework.framework/
 Electron.app/Contents/Frameworks/Electron Framework.framework/Electron Framework
+Electron.app/Contents/Frameworks/Electron Framework.framework/Helpers
 Electron.app/Contents/Frameworks/Electron Framework.framework/Libraries
 Electron.app/Contents/Frameworks/Electron Framework.framework/Resources
 Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/
 Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/
 Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
+Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Helpers/
+Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Helpers/chrome_crashpad_handler
```

Comment:
It's hard-coded in `//components/crash` [[1](https://source.chromium.org/chromium/chromium/src/+/master:components/crash/core/app/crashpad_mac.mm;l=128-129), [2](https://source.chromium.org/chromium/chromium/src/+/master:components/crash/core/app/BUILD.gn;l=217)], so to rename it we'd have to take a patch.

Do you think that's worth it?

Reply:
I would like to stay away from a patch, for one if the name `chrome_` is not friendly then we can attempt a CL in upstream to make the naming more generic like `chromium_` or provide a template to allow embedders provide the executable name. 

Secondly the crashpad_handler shouldn't be messed up at packaging level for any reason, so I don't think the naming is an issue there except for the way it reads :)

Reply:
I'd _prefer_ to drop the `chrome_` part of the name.  I think chromium would take an upstream patch where the name is conditionally `chrome_crashpad_handler` or`crashpad_handler` depending on `is_official_build` or some other chrome specific flag.

Reply:
@MarshallOfSound seems reasonable; I'll take a stab at upstreaming that. If it's accepted this can be updated. If not, I think the extra `chrome_` is fine.

Reply:
(I'll do that as a followup if that's cool 👍 )